helmnet/spectral.py

```diff
@@ -129,11 +148,17 @@ def __init__(self, size: int, direction="x"):
 
     def forward(self, x):
         """x must be [batch, x, y, real/imag]"""
-        return torch.ifft(
-            torch.fft(x, signal_ndim=2, normalized=False).flip(dims=[3]).mul(self.k),
-            signal_ndim=2,
-            normalized=False,
-        )
+        return torch.view_as_real(torch.fft.ifftn(
```

Comment:
Same here. If it is easier to work with complex numbers we could switch to them, and we convert to/from real tensors with two channels only when interfacing with the neural network